reports/RJ722-II.md

```diff
@@ -0,0 +1,439 @@
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/28
+ - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/29
+ - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/30
+ - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/31
+bio: >
+ Rahul is currently a junior, studying Electronics Engineering at Zakir Hussain
+ College of Engineering and Technology, Aligarh Muslim University. He loves the
+ ideology behind Free and Open Source Software and reads about the history of
+ this revolutionary movement in his leisure time.
+social:
+ - GitHub:
+   - username: RJ722
+   - link: https://github.com/RJ722
+ - GitLab:
+   - username: RJ722
+   - link: https://gitlab.com/RJ722
+ - Gitter:
+   - username: RJ722
+   - link: https://gitter.im/RJ722
+ - Openhub:
+   - username: RJ722
+   - link: https://openhub.net/accounts/RJ722
+email: rj722@protonmail.com
+blog: https://rj722.github.io/blog
+activity:
+ - 1:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/b8a8ec8b42b24287560fd07ff3cafe30e92fb585
+   - details: Initial commit
+
+ - 2:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/43f6dae0818cdba691fb3f78e6bb4381867beeb3
+   - details: Create LICENSE
+
+ - 3:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/312313ad96e22386653ca39686a2eedfa00de82c
+   - details: Use README.rst and LICENSE.txt
+
+ - 4:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/d2fd1251e69e2e263645f9189c784a82fec81b4a
+   - details: Add .gitignore
+
+ - 5:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/22fbb2281503fc75b8960df528e878e2cfa24ed3
+   - details: Add docs and news
+
+ - 6:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/7fc63f7097ba83093740b4b3c1eea788fd485dd1
+   - details: Package codet
+
+ - 7:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/680cc58051d3640b73f9ae1ce936193843b9b604
+   - details: Ignore dist/
+
+ - 8:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/111bc5d5cab4c516a7eecb75dd637722f1b1aa37
+   - details: Remove nargs=1
+
+ - 9:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/cbdbd4e391925b69c93d64c794d925422b9a10e6
+   - details: Minor Cosmetics
+
+ - 10:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/f8d854059f740d14147dc92906842e5c1bc04c34
```

Comment:
wouldnt have made it through any review; should have been squashed into a proper commit